### PR TITLE
Add Opera versions for TextMetrics API

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR adds real values for Opera and Opera Android for the `TextMetrics` API, based upon results from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v1.1.6).  Results are manually confirmed for accuracy.

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/TextMetrics

### CODE REVIEW

File: api/TextMetrics.json [hunk 1]
```diff
@@ -602,10 +602,10 @@
               "version_added": false
             },
             "opera": {
-              "version_added": null
+              "version_added": "73"
```
Comment (foolip) [inline api/TextMetrics.json:605]: The Chrome data suggests this is behind a flag and thus not shipped. Can you update that too if it's not correct?

File: api/TextMetrics.json [hunk 2]
```diff
@@ -602,10 +602,10 @@
...
             },
             "opera_android": {
-              "version_added": null
+              "version_added": false
```
Comment (foolip) [inline api/TextMetrics.json:608]: I'd be surprised if this API was different between desktop and mobile, can you verify this isn't in Chrome for Android? Then I'd be more inclined to trust this.

File: api/TextMetrics.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/TextMetrics.json:605]: #7989 adds the Chrome data, which adds versions for when it was no longer behind a flag.  Realize I should have put these changes with that PR, sorry!

File: api/TextMetrics.json (hunk 2 above)
Comment (vinyldarkscratch) [reply api/TextMetrics.json:608]: This _was_ added in Chrome Android, enabled by default in Chrome 87.  There's no Opera Android version that corresponds to Chrome 87, though, hence `false`.

Comment (foolip) [reply api/TextMetrics.json:608]: Oh, I see, thanks!

File: api/TextMetrics.json (hunk 1 above)
Comment (foolip) [reply api/TextMetrics.json:605]: Aha, so Chrome 87, which is the same as Opera 73. Thanks!

**Review by foolip — APPROVED** [review]

**Pull request merged**